src/components/MainPage.js

```diff
@@ -106,8 +122,11 @@ class MainPage extends Component {
 				});
 				this.setState({ dataSet: dailyDataSet });
 			})
+				.then(() => {
+					scroll.scrollTo(325, {smooth: true})
```

Comment:
For a **desktop user** this will move the user to this location so they can see the Graph now that it is populated.